src/custom-element.js

```diff
@@ -1529,12 +1529,11 @@ function createBaseCustomElementClass(win) {
     /**
      * Turns the loading indicator on or off.
      * @param {boolean} state
-     * @param {{cleanup:boolean,force:boolean}=} opt_options
+     * @param {{cleanup:boolean}=} opt_options
      * @public @final @this {!Element}
      */
     toggleLoading(state, opt_options) {
       const cleanup = opt_options && opt_options.cleanup;
-      const force = opt_options && opt_options.force;
```

Comment:
`force` seems to be a bad pattern we've added because we mixed the responsibility of "create loading element" with "show the loading element".

Splitting them seems like the best option (not to mention it'll clean up `toggleLoading`'s code).